packages/flutter_tools/lib/src/devtools_launcher.dart

```diff
@@ -56,20 +56,16 @@ class DevtoolsServerLauncher extends DevtoolsLauncher {
       bool offline = false;
       bool useOverrideUrl = false;
       try {
-        Uri uri;
-        if (_platform.environment.containsKey(_pubHostedUrlKey)) {
-          useOverrideUrl = true;
-          uri = Uri.parse(_platform.environment[_pubHostedUrlKey]);
-        } else {
-          uri = Uri.https('pub.dev', '');
-        }
+        final Uri uri = Uri.parse('http://pub.dev');
         final io.HttpClientRequest request = await _httpClient.headUrl(uri);
         final io.HttpClientResponse response = await request.close();
         await response.drain<void>();
         if (response.statusCode != io.HttpStatus.ok) {
+          _logger.printError('pub offline: ${response.statusCode}');
```

Comment:
I believe in order to see these in the test log you also need to set `debug: true` on the `runFlutter` call in the the test.

Reply:
Actually, looks like they made there way into the log, but the exception is the same:

```
stderr: pub offline: HandshakeException: Handshake error in client (OS Error: 
stderr: 	CERTIFICATE_VERIFY_FAILED: unable to get local issuer certificate(../../third_party/boringssl/src/ssl/handshake.cc:359))
```

From https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8848501694928033008/+/u/run_test.dart_for_tool_integration_tests_shard_and_subshard_4_5/stdout